nft-marketplace/app/ui/common/NavBar.jsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import { Link } from "react-router-dom";
+import { RoutePaths } from "./Routes";
+import { Disclosure } from '@headlessui/react'
+import { MenuIcon, XIcon } from '@heroicons/react/outline'
+
+const navigation = [
+  { name: 'Nav Link 1', href: '#' },
+  { name: 'Nav Link 2', href: '#' },
+  { name: 'Nav Link 3', href: '#' },
+];
+
+export const NavBar = () => {
+  return (
+    <Disclosure as="nav" className="bg-white-lilac">
+      {({ open }) => (
+        <>
+          <div className="max-w-7xl mx-auto px-2 sm:px-6 lg:px-8">
+            <div className="relative flex items-center justify-between h-16">
+              <div className="flex-1 flex items-center justify-between sm:items-stretch">
+                <div className="flex-shrink-0 flex items-center">
+                  <Link to={RoutePaths.ROOT}><h1>NFT Marketplace</h1></Link>
+                </div>
+                <div className="hidden sm:block sm:ml-6">
+                  <div className="flex items-center space-x-4">
+                    {navigation.map((item, index) => (
+                      <Link className="px-3 py-2 rounded-md text-sm font-medium" to={item.href} key={index}>
+                        {item.name}
```

Comment:
make sure to not use `index` as a key
https://reactjs.org/docs/lists-and-keys.html